luigi/rpc.py

```diff
@@ -24,6 +24,7 @@
 import logging
 import socket
 import time
+from urlparse import urljoin
```

Comment:
Yea, you can iterate locally to catch these errors early @paxan, see [CONTRIBUTING](https://github.com/spotify/luigi/blob/master/CONTRIBUTING.rst) file. :)